eq-author-api/schema/resolvers/logic/routing2/expressionGroup2/index.js

```diff
@@ -7,6 +7,28 @@ const { getPages } = require("../../../utils");
 
 Resolvers.ExpressionGroup2 = {
   expressions: expressionGroup => expressionGroup.expressions,
+  validationErrorInfo: ({ id }, args, ctx) => {
+    const expressionGroupErrors = ctx.validationErrorInfo.filter(
+      ({ expressionGroupId }) => expressionGroupId === id
+    );
+
+    if (!expressionGroupErrors) {
+      const noErrors = {
+        id,
+        errors: [],
+        totalCount: 0,
+      };
+      return noErrors;
+    }
```

Comment:
This is also dropping the coverage

Reply:
Weird, I've put a test in for that. I'll take a look.

Reply:
Had a look and I'm not sure why this isn't being covered. The test is in `schema/tests/routing.test.js`